ivadomed/losses.py

```diff
@@ -2,14 +2,36 @@
 import torch.nn as nn
 
 
+# Inspired from https://arxiv.org/pdf/1802.10508.pdf
+class MultiClassDiceLoss(nn.Module):
+    """
+    :param classes_of_interest:  list containing the index of a class which dice will be added to the loss.
+    """
+    def __init__(self, classes_of_interest):
+        super(MultiClassDiceLoss, self).__init__()
+        self.classes_of_interest = classes_of_interest
+
+    def forward(self, prediction, target):
+        dice_per_class = 0
+        n_classes = prediction.shape[1]
+
+        if self.classes_of_interest is None:
+            self.classes_of_interest = range(n_classes)
+
+        for i in self.classes_of_interest:
+            dice_per_class += dice_loss(prediction[:, i, ], target[:, i, ])
```

Comment:
Does `prediction[:, i, ]` really works ? (i am a bit surprised about the empty third index) :)